core/src/main/java/greencity/controller/ClientController.java

```diff
@@ -64,10 +65,11 @@ public ResponseEntity<List<OrderClientDto>> getAllOrdersDoneByUser(
         @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
         @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
     })
-    @GetMapping("/get-all-orders-data/{lang}")
-    public ResponseEntity<List<OrderStatusForUserDto>> getAllDataForOrder(
-        @ApiIgnore @CurrentUserUuid String uuid, @PathVariable Long lang, @ApiIgnore Pageable page) {
-        return ResponseEntity.status(HttpStatus.OK).body(ubsClientService.getOrdersForUser(uuid, lang, page));
+    @GetMapping("/get-user-orders")
```

Comment:
avoid using "get" in URL name. read this: https://restfulapi.net/resource-naming/